src/photos/components/Photo.jsx

```diff
@@ -26,7 +25,7 @@ const getStyleFromBox = box => {
   return style
 }
 
-const Photo = props => {
+const Photo = (props, { client }) => {
```

Comment:
I can't understand how it is related to cozy-to-cozy sharing.

Reply:
FYI, I cherry-picked the commit and merged it in another PR